asn1ate/sema.py

```diff
@@ -719,6 +721,18 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self,elements):
+	self.module2symbols = { }
```

Comment:
All that I use it for is to extract the modules that need to be loaded first, for dependencies of definitions.
I did not cover the full semantics of IMPORTS and EXPORTS because they are very complex.